README.md

```diff
@@ -10,17 +10,16 @@ ember install ember-cli-lazysizes
 
 ## Usage
 
-Usage is simple, all you have to do is to assign the class (default: `'lazyload'`) to image tag in your template:
+Usage is simple, all you have to do is to assign the class (default: `'lazyload'`) to any image tag in your template,
+and use `data-src` or `data-srcset` attributes instead of `src` or `srcset`:
 
 ```html
-<img src="./tomster.png" alt="tomster" class="lazyload">
+<img data-src="tomster.png" alt="tomster" class="lazyload">
```

Comment:
I think the example was incorrect, as using `src` would make the browser load the image immediately, not lazy, right?

Reply:
Exactly, good catch.. Should also change this in our [demo app](https://github.com/kaliber5/ember-cli-lazysizes/blob/master/tests/dummy/app/templates/application.hbs).